c/include/nnstreamer.h

```diff
@@ -1263,6 +1263,24 @@ int ml_tensors_data_set_tensor_data (ml_tensors_data_h data, unsigned int index,
  */
 int ml_check_nnfw_availability (ml_nnfw_type_e nnfw, ml_nnfw_hw_e hw, bool *available);
 
+/**
+ * @brief Checks the availability of the given execution environments.
```

Comment:
It's the same as L1251.
```suggestion
 * @brief Checks the availability of the given execution environments with custom option.
```

Reply:
updated